doc/api/util.md

```diff
@@ -10,6 +10,67 @@ module developers as well. It can be accessed using:
 const util = require('util');
 ```
 
+## util.callbackify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function} An `async` function
+* Returns: {Function} a callback style function
+
+This method converts Promise returning API endpoints to ones that use callbacks.
```

Comment:
Instead of API endpoints, can you just say functions? API endpoints may cause people to think of things like HTTP endpoints.